packages/beacon-node/package.json

```diff
@@ -99,9 +99,10 @@
     "@chainsafe/as-sha256": "^0.3.1",
     "@chainsafe/bls": "7.1.1",
     "@chainsafe/blst": "^0.2.9",
-    "@chainsafe/discv5": "^5.1.0",
-    "@chainsafe/libp2p-gossipsub": "^10.1.1",
-    "@chainsafe/libp2p-noise": "^13.0.1",
+    "@chainsafe/discv5": "^7.1.0",
+    "@chainsafe/enr": "^2.0.2",
+    "@chainsafe/libp2p-gossipsub": "^11.0.1",
```

Comment:
@wemeetagain  is it worth to use v11.1.0 for this upgrade? that's for `batchPublish` option https://github.com/ChainSafe/js-libp2p-gossipsub/pull/482

Reply:
Yeah, we can upgrade it to 11.1.0. Should we turn on batchPublish in certain places?

Reply:
yes just need to add a new option to `Eth2Gossipsub` constructor `batchPublish: true`. That PR need to release first

Reply:
found some weird metrics with `batchPublish` option so let's not do it in this PR. "Validator miss ratio" is so high and balance delta is so high too (first half is with `batchPublish` as true and 2nd half is false)

<img width="851" alt="Screenshot 2024-01-16 at 20 19 40" src="https://github.com/ChainSafe/lodestar/assets/10568965/3f84adbd-2f78-48f5-952d-a09e9a82336c">